### PR TITLE
Renaming and endpoint configuration

### PR DESCRIPTION
**Pull request by aubcel**
* `mapKeys`: renaming to stick to clockodos variable names
* `api`: add optional `baseUrl` with default value

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -379,9 +379,9 @@ Get individual Clockodo Task by its ID.
 clockodo
     .getTaskDuration(
         {
-            taskCustomerId: 23,
-            taskProjectId: 25,
-            taskServiceId: 42,
+            taskCustomersId: 23,
+            : 25,
```
Comment (jhnns) [inline README.md:383]: Hier fehlt der Key:

```suggestion
            taskProjectsId: 25,
```

File: src/internals/api.ts [hunk 2]
```diff
@@ -84,7 +85,11 @@ export class Clockodo {
             throw new Error("Clockodo cacheTime expected to be a number, is typeof: " + typeof cacheTime);
         }
 
-        this[clockodoApi] = new ClockodoLib({user, apiKey, cacheTime});
+        if (baseUrl !== undefined && typeof baseUrl !== "string") {
```
Comment (jhnns) [inline src/internals/api.ts:88]: Sollte ausreichen:

```suggestion
        if (typeof baseUrl !== "string") {
```

File: src/internals/utilities/lib.ts [hunk 3]
```diff
@@ -23,9 +23,9 @@ const transformRequestOptions = params => {
 };
 
 export class ClockodoLib {
-    constructor({user, apiKey, cacheTime}: { user: string; apiKey: string; cacheTime?: number }) {
+    constructor({user, apiKey, cacheTime, baseUrl = ENDPOINT}: { user: string; apiKey: string; cacheTime?: number; baseUrl: string }) {
```
Comment (jhnns) [inline src/internals/utilities/lib.ts:26]: This is just a small thing, but I was wondering why the default base url is called ENDPOINT? Why different names for the same thing? Personally I would prefer DEFAULT_BASE_URL as the name suggests what it is.

File: src/internals/api.ts (hunk 2 above)
Comment (aubcel) [reply src/internals/api.ts]: Well I merged it, but actually the `baseUrl` can be `undefined` in the `api.ts`. And if it is `undefined` it shouldnt' throw an error, which would happen with the solution above.

**Pull request merged**